src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -37,15 +37,20 @@ class SlowOps final : public DaemonHealthMetricCollector {
     if (daemons.empty()) {
       return;
     }
-    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec";
-    check.summary = boost::str(boost::format(fmt) % value.n1 % value.n2);
+    static const char* fmt = "%1% slow ops, oldest one blocked for %2% sec, %3%";
     ostringstream ss;
     if (daemons.size() > 1) {
-      ss << "daemons " << daemons << " have slow ops.";
+      if (daemons.size() > 10) {
+        ss << "daemons " << vector<DaemonKey>(daemons.begin(), daemons.begin()+10)
```

Comment:
Let's only output the ellipsis if we have really spilled beyond the 10 item limit, so that if there's a shorter list people know they're seeing the complete set of services.

Reply:
@jcsp Can you clarify? Doesn't this code do just that?